app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -22,25 +22,27 @@ def create_content(file, relation = :original_file)
         file_set.label ||= label_for(file)
         file_set.title = [file_set.label] if file_set.title.blank?
         return false unless file_set.save # Need to save to get an id
-        IngestJob.perform_later(wrapper!(file, relation))
+        IngestJob.perform_later(wrapper!(file: file, relation: relation))
       end
 
       # Spawns asynchronous IngestJob with user notification afterward
       # @param [Hyrax::UploadedFile, File, ActionDigest::HTTP::UploadedFile] file the file uploaded by the user
       # @param [Symbol, #to_s] relation
       # @return [IngestJob] the queued job
       def update_content(file, relation = :original_file)
-        IngestJob.perform_later(wrapper!(file, relation), notification: true)
+        IngestJob.perform_later(wrapper!(file: file, relation: relation), notification: true)
       end
 
       # Spawns async ImportUrlJob to attach remote file to fileset
       # @param [#to_s] url
       # @return [IngestUrlJob] the queued job
+      # @todo Remove as it appears to be untested and not called
```

Comment:
Looks like it was introduced here https://github.com/samvera/sufia/commit/26eed26343bf41a321bef125c4a8f39106f78bd8 for S3/Fog imports.  So I think you're right that this is no longer necessary.